manifest/schema1/manifest_test.go

```diff
@@ -79,7 +86,7 @@ func genEnv(t *testing.T) *testEnv {
 
 	name, tag := "foo/bar", "test"
 
-	m := Manifest{
+	invalid := Manifest{
```

Comment:
Ah sorry, the FSLayers entry in this initialization was folded up and I didn't notice.